jsvpsa/features/site_owner.feature

```diff
@@ -26,6 +34,20 @@ Feature: Site Owner
     And I should be on "foo"
 
   @api @javascript @dev @content
+  Scenario: Add Private Page
```

Comment:
Protip: You probably don't need to go through the whole content creation portion here as that should be tested in the feature for the module itself. You probably just want to test access to it:

```
Then I should be able to edit a/an :type( content)
```